### PR TITLE
WebSocket Server is not closing properly!!

### PR DESCRIPTION
**Pull request by Nathan-ma**
I had to change a few things in order to make this lib work on esp8266 on Async mode using platformio.

But the main reason on this pull request is the mistake on line 916 on WebSocketsServer, the function is calling itself in a eternal loop that will end up eventually breaking the code.

Fix is simple, instead of WebSocketServer::close(), just reference to the proper one WebSocketServerCore::close().

### CODE REVIEW

File: src/WebSocketsServer.cpp [hunk 1]
```diff
@@ -913,7 +913,7 @@ void WebSocketsServer::begin(void) {
 }
 
 void WebSocketsServer::close(void) {
-    WebSocketsServer::close();
+    WebSocketsServerCore::close();
```
Comment (d-a-v) [inline src/WebSocketsServer.cpp:916]: I apologize for the mistake !

Comment (Nathan-ma) [reply src/WebSocketsServer.cpp:916]: Glad I could Help!

**Comment by Links2004** [comment]
ok the github build in conflict resolver bad. changes all line space and line endings...

**Comment by Links2004** [comment]
changes have been manually merge to master

**Pull request closed without merging**